README.md

```diff
@@ -1 +1,241 @@
-# Bookstore
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
```

Comment:
- [ ] Kindly remove this commented code from your project. These are guides and should be removed when the code has been implemented. It keeps your code clean, and  professional 👍